Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -128,25 +128,15 @@ private void GenerateNodeGrid()
 
             //Info(point.ToString());
             Collider2D[] cols = Physics2D.OverlapCircleAll(point, 0.25f, LayerMask.GetMask("Ship", "ShortObjects"));
-            List<Collider2D> colsList = cols.Where(col => !col.isTrigger && !col.transform.name.Contains("Vent") && !col.transform.name.Contains("Door")).ToList();
+            List<Collider2D> colsList = cols
+                .Where(col =>
+                    !col.isTrigger
+                    && !col.transform.name.Contains("Vent")
+                    && !col.transform.name.Contains("Door")
+                    && !col.transform.name.Equals("Collider") // The Mira decontamination doors have the default name for some reason. No other map has colliders with the default name.
```

Comment:
Instead of checking for the name, use GetComponent to check for whatever components are on that object.

Reply:
Could you elaborate a bit on this? How would *you* go about finding what Component types I should be searching for, because I can't seem to find anything in the Assembly-CSharp.

Reply:
If GetComponent (maybe GetComponentInParent but not sure) of type `Vent` or `SomeKindaDoor`

Reply:
So I've discovered the hierarchy for the decontam doors, which also explains the poor collider name.
![image](https://user-images.githubusercontent.com/72096833/228677900-5df760a3-4c7d-4182-9672-4a10ccfa9ac6.png)

I don't think this browser program will tell me the components on the objects, but knowing the hierarchy should be good enough.

Reply:
As I said in my previous message, the components are `Vent` and `SomeKindaDoor`